nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -590,4 +590,37 @@ $(XamlMetaDataProviderPch)
         </Link>
     </ItemDefinitionGroup>
 
+    <!--
+    ============================================================
```

Comment:
There are a few too many negatives in this sentence. Consider rewriting for clarity.

Reply:
I think there's a typo (which would remove one negative):  "Note that we..."